test-tools/wamr-ide/VSCode-Extension/src/utilities/directoryUtilities.ts

```diff
@@ -94,3 +95,24 @@ export function CheckIfDirectoryExist(path: string): boolean {
         return false;
     }
 }
+
+export function checkFolderName(folderName: string) {
+    var status = true;
+    let OS_PLATFORM = os.platform();
+    if (folderName.length > 255) {
+        status = false;
+    }
+    let errArr: string[] = [];
+    if (OS_PLATFORM === 'win32') {
+        errArr = ['\\', '/', ':', '?', '*', '"', '|', '<', '>'];
+    } else if (OS_PLATFORM === 'linux') {
```

Comment:
add `|| OS_PLATFORM === 'darwin'`

or, maybe it's simpler to reject any characters which are invalid on any of platforms.

Reply:
Thanks for the comment, updated